src/UI/Root/Components/Header/EnvSelector/EnvSelector.tsx

```diff
@@ -45,30 +44,20 @@ export const EnvSelector: React.FC<Props> = ({
           aria-label={toggleText}
           isFullHeight
           onClick={() => setIsOpen(!isOpen)}
-          icon={authController.isEnabled() ? <UserCircleIcon /> : null}
+          icon={!authHelper.isDisabled() ? <UserCircleIcon /> : null}
         >
-          {authController.isEnabled() ? (
-            <StyledDiv>
+          <StyledDiv>
+            <div>
+              {!authHelper.isDisabled() ? (
```

Comment:
{!authHelper.isDisabled() && <StyledText>{authHelper.getUser()}</StyledText>}
      
There's no need for the ternary here. You changed the wrong line, there was nothing wrong with line 47.